engine/runtime/src/main/java/org/enso/interpreter/node/expression/literal/PatchableLiteralNode.java

```diff
@@ -0,0 +1,77 @@
+package org.enso.interpreter.node.expression.literal;
+
+import com.oracle.truffle.api.frame.VirtualFrame;
+import com.oracle.truffle.api.nodes.Node;
+import com.oracle.truffle.api.nodes.NodeInfo;
+import com.oracle.truffle.api.source.SourceSection;
+import java.util.function.Predicate;
+import org.enso.compiler.core.IR;
+import org.enso.compiler.core.IR$Literal$Number;
+import org.enso.compiler.core.IR$Literal$Text;
+import org.enso.interpreter.node.ExpressionNode;
+import org.enso.interpreter.runtime.data.text.Text;
+import org.enso.interpreter.runtime.tag.Patchable;
+
+/** Generic literal node. */
+@NodeInfo(shortName = "Literal", description = "Constant literal expression")
```

Comment:
Is this the corect `shortName`. Please document the difference between `LiteralNode` and `PatchableLiteralNode` and when one should use one or the other (or whether it should be created explicitly at all because it seems no)

Reply:
Yes, I can see that it is in the PR description but not in javadoc.